subprojects/detekt.yml

```diff
@@ -298,8 +298,9 @@ formatting:
     NoMultipleSpaces:
         active: true
         autoCorrect: false
+    # https://detekt.github.io/detekt/formatting.html#nosemicolons
```

Comment:
I'd great to see a link to the documentation with examples: https://ktlint.github.io/#rule-semi

Reply:
will change links to ktlint